Automattic/wp-calypso
# WordAds: Prevent Crash & Hide Tab in Stats for Non-Admins

#### Changes proposed in this Pull Request

The "Ads" tab is currently not hidden for non-admins. This crashes the screen if clicked by a non-admin.

<img width="1866" alt="Screenshot 2019-07-14 at 20 31 46" src="https://user-images.githubusercontent.com/43215253/61188408-6dccc980-a676-11e9-8a1a-a072623d32e3.png">

#### Testing instructions

_I'd recommend to hide whitespaces when checking the code!_

On a site with WordAds, visit `/stats`, and verify that for non-admins "Ads" no longer appears.

<img width="314" alt="Screenshot 2019-07-14 at 20 32 35" src="https://user-images.githubusercontent.com/43215253/61188415-8937d480-a676-11e9-99c9-896b172efcb5.png">

**Visiting the route directly**

If the user is not an admin, this should appear.

<img width="1269" alt="Screenshot 2019-07-16 at 17 53 44" src="https://user-images.githubusercontent.com/43215253/61313600-abeef800-a7f2-11e9-879d-d351975f9b9a.png">

If the user is an admin (and therefore can upgrade to WordAds), this should appear. The CTA redirects to` /earn/ads-settings/site`

<img width="984" alt="Screenshot 2019-07-16 at 17 54 02" src="https://user-images.githubusercontent.com/43215253/61313648-c5903f80-a7f2-11e9-95e2-8dd901dcf588.png">

If the user has WordAds and is an admin, the page should load as normal.

Lint errors are unrelated. :)

Fixes an issue I noticed whilst working on #34597
